### DISCUSSION TITLE
some question about the paper

### DISCUSSION THREAD

**Discussion by ninjaxfy** [GoekeLab/proActiv#36] [Q&A] [answered]
Sorry to bother!
I 'd appreciate a lot if you can reply !!

I have read your paper , which is an excellent work. However, I might not comprehend the paper very well, and do have some question in the **CAGE Tag Analysis part**. 
My question is that the paper has showed that **the inactive promoters showed low percentage of CAGE tag support**. And CAGE-seq data is thought as the golden standard of the transcriptom start sites, hence **whether inactive promoters' start site called by proActiv was less possible to be real transcriptom start site**?
**And I tried to apply the proActiv to other species, the result showed low percentage of overlapping with CAGE-seq data.**

If it is available, please reply. Your reply will do me a lot favor, appreciate again!
 
Best wishes!!!

**Comment by jonathangoeke** [comment, marked as answer]
Hi @ninjaxfy all promoters used by proActiv are annotated, whether they are active or not does not mean they are real or not (they could for example be active in a different tissue). Inactive promoters show low levels of transcription, similar to start sites with low CAGE tag support. I hope that helps!
